src/handler.ts

```diff
@@ -16,7 +16,9 @@ export interface Config {
   skipUsers: string[]
 }
 
-export async function handlePullRequest(context: Context): Promise<void> {
+export async function handlePullRequest(
+  context: Context<'pull_request.opened' | 'pull_request.ready_for_review'>
```

Comment:
Believe we should establish `type PRContext = Context<'pull_request.opened' | 'pull_request.ready_for_review'>` and reference that in the locations in this PR, but may be unnecessary.

Reply:
I like the idea... I'm not sure I could get it to work though. It got a little clunky for me because `Context` is used as a class in `test/handler.ts`:
```
context = new Context(event, {} as any, {} as any)
```

So I (think I) need to declare PRContext as a class... and if I do so, extending Context<>, I'll need to redefine the constructor. I don't think there's a shorthand there, but my TS is rusty..